src/wrapper/java_vm/vm.rs

```diff
@@ -67,7 +67,7 @@ impl JavaVM {
     }
 
     /// Attaches the current thread to a Java VM as a daemon.
```

Comment:
Shall we clarify here in the docs that the AttachGuard will detach the daemon thread when it goes out of the scope? I guess that this operation is mostly used to attach a thread as a daemon for _some_ time — for longer than a single method execution, hence one will have to remember to keep the guard for that period of time somewhere.